src/main/java/frc/bionic/swerve/AbstractDrivetrain.java

```diff
@@ -36,6 +40,14 @@
 
   private double                  kMaxSpeed = 2.0;  // meters per second
 
+  //Shuffleboard Related
+  private String                   name;
```

Comment:
The usage here is different than in the other files. In the other files, `name` is to uniquely identify one of many similar items, e.g., "LR", "RR", etc. for swerve modules... while `shuffleboardTabName` is used to identify which tab to place it on. Since `name` here is being used to identify the tab name, I suggest renaming it (both member name and parameter name below) to `shuffleboardTabName`.